src/test/java/com/incognia/IncogniaAPITest.java

```diff
@@ -298,9 +300,11 @@ void testRegisterLogin_whenAccountIdIsNotValid() {
 
   private void assertTransactionAssessment(TransactionAssessment transactionAssessment) {
```

Comment:
It looks like address evidences are not being validated here :thinking: 
It would be nice to add that validation, although it could be done in a separate PR.

Also, I believe it would be interesting to have tests (for each use case) for when evidences are not available, as some are not enabled by default, such as device-id.

Reply:
I'll add this to the lists of improvements to the lib, and do it in a next PR